src/component/field.js

```diff
@@ -17,10 +17,10 @@ const amountStyles = StyleSheet.create({
   },
 });
 
-export const AmountInputField = ({ style, ...props }) => (
+export const AmountInputField = ({ fiat, style, ...props }) => (
   <HorizontalExpandingTextInput
-    style={[amountStyles.input, style]}
-    charWidth={46}
+    style={[amountStyles.input, { textAlign: fiat ? 'left' : 'right' }, style]}
+    charWidth={40}
```

Comment:
Heads up that the 46 pixels is required for the TextInput width to scale correctly.